tests/test_sampling/test_filterer.py

```diff
@@ -22,14 +22,14 @@ class FiltererTest(unittest_templates.GenericTestCase[Filterer]):
     def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMapping[str, Any]:  # noqa: D102
         kwargs = super()._pre_instantiation_hook(kwargs=kwargs)
         self.generator = set_random_seed(seed=self.seed)[1]
-        kwargs["triples_factory"] = self.triples_factory = Nations().training
+        self.triples_factory = Nations().training
+        kwargs["mapped_triples"] = self.triples_factory.mapped_triples
         return kwargs
 
     def post_instantiation_hook(self) -> None:  # noqa: D102
-        self.slcwa_instances = self.triples_factory.create_slcwa_instances()
-        self.positive_batch = self.slcwa_instances.mapped_triples[torch.randint(
+        self.positive_batch = self.triples_factory.mapped_triples[torch.randint(
             low=0,
-            high=len(self.slcwa_instances),
+            high=self.triples_factory.num_triples,
```

Comment:
For this part, we would not necessesarily need to store the `triples_factory` in an attribute (we could just use the `mapped_triples` directly)